### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards- Import- Weather Card]: Talkbak does not announce complete information about each day(Thursday, Friday) on Weather card.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8472]
### Target Platforms

Android

### SDK Version

2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment
Device: S21+ 5G
Screen Reader: Talkback
Adaptive Card Android App version: 2023.03.31.1

Repro Steps:

1. Launch the Adaptive cards app on Android.
2.  Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Weather card'.
6. Navigate to 'Visual' tab control and select it.
7. Turn on Talkback feature.
8. Navigate on each control under 'Visual' tab and observe the issue.

Actual Result:
Talkbak does not announce complete information about each day(Thursday, Friday) on Weather card. talkback is announcing, "View Thursday".

Expected Result:
Talkback should announce the complete information about the content present for each day as, "View Thursday, Sunshine with clouds Highest temperature 50 degrees lowest temperature 32 degrees".

User Experience:
Users which depends on AI might get confused if Talkback is not announcing the complete information about the content present for each day.

Mas Reference:
[MAS 4.2.1 – Object Information ](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/89897257/232738934-ad910acd-6983-4b52-a130-9de2ed73ea7f.mp4

### Card JSON

```JSON
NA
```
